content/en/docs/concepts/acme-orders-challenges.md

```diff
@@ -0,0 +1,98 @@
+
+In order to complete these challenges, cert-manager introduces two
+`CustomResource` types; Orders and Challenges.
+
+## Orders
+
+`Order` resources used by the ACME issuer to manage the lifecycle of an ACME
+'order' for a signed TLS certificate. An order represents a single certificate
+request which will be created automatically once a new
+[`CertificateRequest`](../certificaterequest/) resource referencing an ACME
+issuer has been created. `CertificateRequest` resources are created
+automatically by cert-manager once a [`Certificate`](../certificate/) resource is
+created, has its specification changed, or needs renewal.
+
+As an end-user, you will never need to manually create an Order resource.
+Once created, an Order cannot be changed. Instead, a new Order resource must be
+created.
+
+The `Order` resource encapsulates multiple ACME 'challenges' for that 'order',
+and as such, will manage one or more `Challenge` resources.
+
+## Challenges
+
+Challenge resources are used by the ACME issuer to manage the lifecycle of an
+ACME 'challenge' that must be completed in order to complete an 'authorization'
+for a single DNS name/identifier.
+
+When an `Order` resource is created, the order controller will create
+Challenge resources for each DNS name that is being authorized with the ACME
+server.
+
+As an end-user, you will never need to manually create a Challenge resource.
+Once created, a Challenge cannot be changed. Instead, a new Challenge resource
+must be created.
+
+### Challenge Lifecycle
+
+After a Challenge resource has been created, it will be initially queued for
+processing. Processing will not begin until the challenge has been 'scheduled'
+to start.  This scheduling process prevents too many challenges being attempted
+at once, or multiple challenges for the same DNS name being attempted at once.
+For more information on how challenges are scheduled, read the challenge
+scheduling. TODO: link to header below for scheduling.
+
+Once a challenge has been scheduled, it will first be 'synced' with the ACME
+server in order to determine its current state. If the challenge is already
+valid, its 'state' will be updated to 'valid', and also set
+`status.processing = false` to 'unschedule' itself.
+
+If the challenge is still 'pending', the challenge controller will 'present' the
+challenge using the configured solver, one of HTTP01 or DNS01.  Once the
+challenge has been 'presented', it will set `status.presented = true`.
+
+Once 'presented', the challenge controller will perform a 'self check' to
+ensure that the challenge has 'propagated' (i.e. the authoritative DNS servers
+have been updated to respond correctly, or the changes to the ingress resources
+have been observed and in-use by the ingress controller).
+
+If the self check fails, cert-manager will retry the self check with a fixed
+10 second retry interval. Challenges that do not ever complete the self check
+will continue retrying until the user intervenes.
```

Comment:
`intervenes by either retrying the Order (by deleting the Order resource) or amending the associated Certificate resource to resolve any configuration errors.`